qubesmanager/restore.py

```diff
@@ -35,8 +34,7 @@
 from . import multiselectwidget
 from . import backup_utils
 
-from multiprocessing import Queue
-from multiprocessing.queues import Empty
+import queue
```

Comment:
`queue.Queue` is different from `multiprocessing.Queue`. According to documentation, the former works with threads, but not necessary with processes. Any specific reason for this change?